index.js

```diff
@@ -0,0 +1,153 @@
+// The Front object is loaded through the Front script added in the header of the main HTML.
+// This object can be used to listen to conversation event data as it occurs on Front, and request information and to perform actions on Front.
```

Comment:
This object can be used to listen to conversation event data as it occurs on Front, request information from Front, and perform actions on Front.